api/configs/app_config.py

```diff
@@ -99,4 +100,7 @@ def settings_customise_sources(
             RemoteSettingsSourceFactory(settings_cls),
             dotenv_settings,
             file_secret_settings,
+            TomlConfigSettingsSource(
+                settings_cls=settings_cls, toml_file=Path(__file__).parent.parent / "pyproject.toml"
```

Comment:
Could we consider implementing a search method to locate the nearest `pyproject.toml` file?

Reply:
Are you looking for a general method that could be reused in dify python plugin SDK?